data/autoyast_opensuse/autoyast_btrfs_quota.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <bootloader>
+        <global>
+            <timeout config:type="integer">-1</timeout>
```

Comment:
Please, also add a section to disable timeout in pop-up messages, so we do not miss unexpected warnings/errors or info messages.

Reply:
Added.